internal/aws/client.go

```diff
@@ -332,6 +333,38 @@ func (c *client) CreateUser(u *User) (user *User, err error) {
 	return
 }
 
+// UpdateUser will update/replace the user specified
+func (c *client) UpdateUser(u *User) (user *User, err error) {
```

Comment:
I would suggest not to use named return parameters. I did not find time to refactor this everywhere.

Reply:
Removed the named parameters